documentation/style-guide.md

```diff
@@ -127,6 +127,7 @@ In code blocks:
     - Used with a `## Codebyte Example` header.
     - Typically, only one per entry.
     - Typically, appears at the end of the entry.
+    - Is original code, not a copy of a prior example.
```

Comment:
```suggestion
    - Is original code, not a copy of a prior example in Docs or an existing example on another site.
```

Reply:
Plagiarism is noted at the top of the document, but I think it may be worthwhile to have some redundancy and use explicit language about the expectations for content.